dll/win32/winmm/time.c

```diff
@@ -422,7 +423,15 @@ MMRESULT WINAPI timeGetDevCaps(LPTIMECAPS lpCaps, UINT wSize)
 MMRESULT WINAPI timeBeginPeriod(UINT wPeriod)
 {
     if (wPeriod < MMSYSTIME_MININTERVAL || wPeriod > MMSYSTIME_MAXINTERVAL)
-	return TIMERR_NOCANDO;
+		return TIMERR_NOCANDO;
+
+	/*  High resolution timer requested, use QPC */
+	if (wPeriod <= 5 && TIME_qpcFreq.QuadPart == 0) {
+		if(QueryPerformanceFrequency(&TIME_qpcFreq))
+			TIME_qpcFreq.QuadPart /= 1000;
+		else
+			TIME_qpcFreq.QuadPart == 0;
```

Comment:
This statement does not do anything. Typo?

Reply:
I see, since there already was a if 0 check above, fixed it.